content/en/hosting/kubernetes-vs-docker.md

```diff
@@ -2,48 +2,118 @@
 title: Kubernetes vs Docker
 linkTitle: Kubernetes vs Docker
 weight: 5
+aliases:
+  -  /hosting/vertical-vs-horizontal
 ---
 
 {{< hextra/hero-subtitle >}}
   Options for installing CHT applications
 {{< /hextra/hero-subtitle >}}
 
-Since the release of CHT Core 4.0.0 in late 2022, Medic has been perfecting the hosting for the toolkit to balance the need for high uptimes so CHWs can always deliver care while having an easy and approachable technical back end hosting solution.  While initially [Docker Compose](https://docs.docker.com/compose/) with an [overlay network](https://docs.docker.com/compose/networking/#multi-host-networking) was thought to be our goto solution, field testing this overlay networks in production has shown them to unreliable.  As such, on this site you will find documentation for both Docker Compose for smaller deployments (with no overlay network) and Kubernetes for larger deployments.
+{{< callout >}}
+Medic previously suggest Kubernetes for all production deployments - this is no longer the case.  Further, Medic no longer suggests multi-node CouchDB.
+{{< /callout >}}
 
-Given all this, we recommend:
-* New, smaller production 4.x CHT deployments can use either [Docker]({{< relref "hosting/4.x/production/docker/" >}}) or [Kubernetes]({{< relref "hosting/4.x/production/kubernetes/" >}}) depending on their expected use and growth.
-* Application development for both [CHT 3.x]({{< relref "hosting/3.x/app-developer" >}}) and [CHT 4.x]({{< relref "hosting/4.x/app-developer" >}}) should use Docker Compose.
-* Production 3.x CHT deployments should use [Docker Compose]({{< relref "hosting/3.x" >}})  - Note that 3.x is [end of life]({{< relref "releases/#supported-versions" >}})  and should only be used to support existing 3.x deployments.
+Medic now recommends [Docker](/hosting/4.x/docker/) for most deployments. Use [Kubernetes](/hosting/4.x/kubernetes/) for multi-tenant or specific hardware constrained deployments. For more details on the research behind suggesting Docker and single-node CouchDB, please see [this forum post](https://forum.communityhealthtoolkit.org/t/investigate-adding-more-shards-as-a-potential-avenue-for-improved-performance/4831?u=mrjones).
```

Comment:
```suggestion
Medic recommends [Docker](/hosting/4.x/docker/) for most deployments. Use [Kubernetes](/hosting/4.x/kubernetes/) for multi-tenant or specific hardware constrained deployments. For more details on the research behind suggesting Docker and single-node CouchDB, see [this forum post](https://forum.communityhealthtoolkit.org/t/investigate-adding-more-shards-as-a-potential-avenue-for-improved-performance/4831?u=mrjones).
```

Reply:
Per the [docs style guide](https://docs.communityhealthtoolkit.org/community/contributing/docs/style-guide/#simple-and-direct-language), we should avoid please.

Reply:
Thanks for the attention to detail - I'll revisit that style guide as a refresher!